mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -222,6 +228,7 @@ CLASS ahm net/minecraft/entity/Entity
 	METHOD b onPlayerCollision (Larz;)V
 	METHOD b interact (Larz;Lagg;)Z
 		ARG 1 player
+	METHOD b vec3dSquaredHorizontalLength (Lcnl;)D
```

Comment:
`squaredHorizontalLength` should be fine here

Reply:
It's a public static method, but fair enough.